components/Logs.tsx

```diff
@@ -22,29 +30,50 @@ export default function Logs<T>({ logs, render, keyer }: LogsProps<T>) {
   }, [logs]);
 
   return (
-    <Box
-      ref={logsElement}
-      bg={colorMode == "dark" ? "steel" : "sunken"}
-      borderRadius="10px"
-      maxWidth="100%"
-      height="500px"
-      overflowY="auto"
-      p={1}
-      fontSize="sm"
-    >
-      {logs.map((log) => (
-        <pre
-          key={keyer(log)}
-          style={{
-            margin: "5px 0",
-            padding: 0,
-            fontSize: "inherit",
-            overflow: "visible",
-          }}
-        >
-          {render(log)}
-        </pre>
-      ))}
-    </Box>
+    <>
+      <FormControl display="flex" alignItems="center">
+        <FormLabel htmlFor="scroll" m="0">
+          Auto Scroll
+        </FormLabel>
+        <Switch
+          id="scroll"
+          isChecked={autoScroll}
+          onChange={() => setAutoScroll(!autoScroll)}
+        />
+
+        <FormLabel htmlFor="wrap" m="0">
+          Word Wrap
+        </FormLabel>
+        <Switch
+          id="wrap"
+          isChecked={wordWrap}
+          onChange={() => setWordWrap(!wordWrap)}
+        />
```

Comment:
style note: this can also be rewritten to have the switch inside the label and remove the IDs (semantically correct as well according to HTML5, not sure how Chakra works with it.) This way works too so I'm not requesting changes.